topologies/datacenter-latest/labguides/source/mlag-with-cvp.rst

```diff
@@ -0,0 +1,116 @@
+MLAG
+====
+
+.. image:: images/cvp-mlag/mlag_1.png
```

Comment:
Could you also add a note in here informing the user to click on a network device to ssh into that device?